gax/src/main/java/com/google/api/gax/retrying/RetrySettings.java

```diff
@@ -200,7 +205,11 @@ public static Builder newBuilder() {
      * <pre>{@code actualDelay = rand_between(0, min(maxRetryDelay, exponentialDelay))}</pre>
      *
      * The default value is {@code true}.
+     *
+     * @deprecated Retries will always jitter.
```

Comment:
Is this true already? I would have expected some updates to concrete methods are also necessary.

Reply:
If we eventually remove the public scoping, then yes. Only [ExponentialRetryAlgorithm](https://github.com/googleapis/gax-java/pull/1308/files#diff-bbb4a04a2c5db9febb7b33ee8d63fcaadadfed01a3f5935f48fcc668dc266d20R190) does anything with this at the moment, but that cannot be changed without removing the test cases mentioned in the PR description.